package.json

```diff
@@ -94,6 +94,7 @@
     "typescript": "5.1"
   },
   "dependencies": {
+    "big-integer": "^1.6.52",
```

Comment:
Why did you choose this over native bigint support?

Reply:
Because generating RSA parameters of size=2048 bits exceed memory limit when operating with native bigints.
Let me tag you on Discord to provide more context on the PR.